jobs/stream.yaml

```diff
@@ -6,11 +6,21 @@ nr_threads:
   - 100%
   - 50%
   - 25%
+  - 12.5%
+  - 6.25%
+
+iterations:
+  - 10x
 
 stream:
   array_size:
     - 10000000 # 0.2GB
     - 50000000 # 1GB
+    - 128000000 # 3GB
+    - 192000000 # 4.5GB
```

Comment:
for the newly add array_size, is it possible to maintain this in your own version of job yamls, so we still keep the default in lkp-tests.

Reply:
The change is updated according to the review comment.